adr/2022-08-decouple-from-ietf.md

```diff
@@ -122,9 +122,10 @@ we don't have the same unpleasant history with W3C than we do with IETF.
 
 * Not being associated with a recognized standards organization such as IETF,
   W3C, or IEEE reduces the credibility of JSON Schema in the eyes of some.
-  However, people seem comfortable adopting OpenAPI without it being associated
-  with a standards organization, so we don't expect this to be a significant
-  issue for JSON Schema either.
+  However, we have received feedback from people involved in standards
+  development that say they are comfortable adopting OpenAPI without it being
+  associated with a standards organization, so we don't expect this to be a
```

Comment:
This was said in the context of the connection with the Linux Foundation, not simply OpenAPI as a completely independent entity.  Which still works as we are also connected.  But it's important to keep the focus on connection to *some* sort of larger organization here.

Reply:
It didn't appear to me that the answer to OpenAPI's approach being acceptable was related to or contingent on their connection with the Linux Foundation. Those appeared to be two separate answers to two separate questions. Considering that the Linux Foundation has nothing to do with standards development or publishing, it's hard to see how the two are related. However, I do see how association with the Linux Foundation provides credibility in the general sense and is therefore worth mentioning in this context.

Reply:
>  However, I do see how association with the Linux Foundation provides credibility in the general sense and is therefore worth mentioning in this context.

Yes, that's what I was trying to get at, thank you for figuring that out despite my poor articulation!

Reply:
> This was said in the context of the connection with the Linux Foundation, not simply OpenAPI as a completely independent entity. Which still works as we are also connected. But it's important to keep the focus on connection to _some_ sort of larger organization here.

Agreed Henry.  Thanks for calling out this point.  It's an important aspect of how things will be perceived as JSON schemas is utilized as a dependency in other standards.